api/tests/unit/environments/test_unit_environments_models.py

```diff
@@ -180,7 +180,7 @@ def test_environment_get_environment_document(environment, django_assert_num_que
     # Given
 
     # When
-    with django_assert_num_queries(4):
+    with django_assert_num_queries(3):
```

Comment:
As above, do we know why we've dropped a query here? (and below in `test_environment_get_environment_document_with_caching_when_document_not_in_cache`)

Reply:
See [this comment](https://github.com/Flagsmith/flagsmith/pull/2290#discussion_r1242489443).